src/adapters/api-blueprint.es6

```diff
@@ -0,0 +1,39 @@
+import Drafter from 'drafter';
+
+// Auto-detect via the API Blueprint format metadata.
+const FORMAT1A = /^[\uFEFF]?(((VERSION:( |\t)2)|(FORMAT:( |\t)(X-)?1A))\n)/i;
+
+export const name = 'api-blueprint';
+export const mediaTypes = [
+  'text/vnd.apiblueprint'
```

Comment:
I think we have a few more `.` after `apiblueprint`. Like `vnd.apiblueprint.markdown`